### ISSUE TITLE
Contrast with procrastinate + migrations

### ISSUE THREAD

**Issue by mahmoud** [LaunchPlatform/bq#1]
Hey there! Great project here, big fan of this architecture.

When comparing Python Postgres queues, there's one project conspicuously absent from the Alternatives list, `procrastinate`: https://github.com/procrastinate-org/procrastinate

That said, I think `bq` compares favorably. Sure, `procrastinate` is more mature and may have more features, but it also focuses more on Django and less on SQLAlchemy, and has [a much more murky migration story](https://procrastinate.readthedocs.io/en/stable/howto/production/migrations.html). If I'm understanding correctly, `bq` should "just work" with Alembic, especially if one creates their own Task models.

So in short:

1. Great work
2. Maybe add Procrastinate to the list of Alternatives
3. Consider adding a short discussion of migrations for when a `bq` upgrade might change its internal model definitions.

Thanks!
